components/scheduler/views/day.md

```diff
@@ -10,7 +10,7 @@ position: 1
 
 # Day View
 
-The Day view of the Scheduler for Blazor shows a single day to the user.
+The Day view of the Scheduler for Blazor shows a single day to the user. You can also configure the Day view to display events [grouped by resource]({%slug scheduler-resource-grouping%}).
```

Comment:
```suggestion
The Day view of the Scheduler for Blazor shows a single day to the user.
```
This gives the impression that the customers need to somehow configure the view to display grouped events which is not true. The resource grouping is configured globally for the component and is not related to the views.